src/leiningen/new/db.clj

```diff
@@ -8,11 +8,18 @@
     (some #{"+mongodb"} features) :mongo
     (some #{"+h2"} features) :h2))
 
+(defn select-sql-dsl [{:keys [features]}]
+  (cond
+    (some #{"+yesql"} features) :yesql
+    (some #{"+korma"} features) :korma))
```

Comment:
I would just default to yesql so users don't have to add 2 flag every time when selecting the db